### PR TITLE
Add template FoolSlideParser ands sources

### LINKED ISSUE #67: Sora Manga (Arabic)
### Source name

Sora Manga

### Source link

https://soramanga.com/

### Language

Arabic

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

### LINKED ISSUE #102: VCP srouce request (+18 content)
### Source name

vercomicsporno

### Source link

https://vercomicsporno.com/comics-porno

### Language

Spanish site (Spanish+English content)

### Other details

Vercomicsporno: https://vercomicsporno.com/comics-porno
Has a new alternative site with different name but same content and specifications: Toonx:
https://toonx.net/comics-porno. If possible please add support for one of both. Thanks so much for your work.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

### LINKED ISSUE #230: TumMangaOnline is not working
### Source information

TuMangaOnline

### Steps to reproduce

When trying to load any chapter it does not work and this message appears: Syntax error in regexp pattern near index 29 \{uniqid:'(.+)', cascade:(.+)}
Screenshot_20230810_080524

### Kotatsu version

5.3.10

### Android version

13

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

### PR DESCRIPTION
**Pull request by davvarrr**
Add template FoolSlideParser ands sources

 fix TuMangaOnlineParser -> close #230 

Sources : 
reader.deathtollscans.net
reader.mangatellers.gr
reader.evilflowers.com
reader.silentsky-scans.net
www.menudo-fansub.com
hni-scantrad.com
reader.powermanga.org
www.ramareader.it
read-nifteam.info
reader.onepiecenakama.pl
birdmanga -> close #67 ( the site given by the issue links to this url ) 
templescanesp.com
tenkaiscan.net
vercomicsporno.com -> close #102 
vermangasporno.com

### CODE REVIEW

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/vi/HentaiVNParser.kt [hunk 1]
```diff
@@ -33,23 +40,26 @@ class HentaiVNParser(context: MangaLoaderContext) : MangaParser(context, MangaSo
...
-			.selectFirstOrThrow("div.page-info")
+
+		val id = docs.location().substringAfterLast("/").substringBefore("-")
+
+		val infoEl = webClient.httpGet("/list-info-all-mobile.php?id_anime=$id".toAbsoluteUrl(domain)).parseHtml()
+
+		val genreUrl =  Regex(""""(list-info-theloai-mobile\.php?.+)"""").find(docs.toString())?.groupValues?.get(1)
+		val genre = webClient.httpGet("https://$domain/$genreUrl").parseHtml().select("a.tag")
+
+		val stateDoc = webClient.httpGet("/list-info-time-mobile.php?id_anime=$id".toAbsoluteUrl(domain)).parseHtml()
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/vi/HentaiVNParser.kt:51]: These calls can be managed in parallel using `async()`

Comment (davvarrr) [reply src/main/kotlin/org/koitharu/kotatsu/parsers/site/vi/HentaiVNParser.kt]: Thanks for the correction, it was added.

Comment (Koitharu) [reply src/main/kotlin/org/koitharu/kotatsu/parsers/site/vi/HentaiVNParser.kt]: Uh, calling `await` right after `async` has no sense, right?)

I'll fix it by my own

Comment (davvarrr) [reply src/main/kotlin/org/koitharu/kotatsu/parsers/site/vi/HentaiVNParser.kt]: Sorry, I thought it worked differently, but thanks for changing it.

**Pull request merged**